src/UCTSearch.h

```diff
@@ -73,12 +73,12 @@ class UCTSearch {
     static constexpr passflag_t NORESIGN = 1 << 1;
 
     /*
-        Maximum size of the tree in memory. Nodes are about
-        48 bytes, so limit to ~1.2G on 32-bits and about 5.5G
-        on 64-bits.
+        Maximum size of the tree in memory.
+        Nodes + Scores average less than 20 bytes.
+        limit to ~1.1G on 32-bits and about 5.6G on 64-bits.
     */
     static constexpr auto MAX_TREE_SIZE =
-        (sizeof(void*) == 4 ? 25'000'000 : 100'000'000);
+        (sizeof(void*) == 4 ? 60'000'000 : 300'000'000);
```

Comment:
60 million nodes for 32-bit, i.e. 4GB max RAM?

Reply:
The original number was set to use ~ 1.2G of ram (see the now updated comment above).

Reply:
Those are **nodes**, max_tree_size counts both expanded and unexpanded nodes.
With 250,000 playouts, tree reuse, and timemanage off you'll probably see 400,000 playouts after some sente move.
~400,000 nodes * ~300 scores/node = 120,000,000 (nodes+scores) * 20 bytes = 2.4 GB


 there (360-move)